services/datamanager/builtin/capture/config.go

```diff
@@ -1,5 +1,17 @@
 package capture
 
+// MongoConfig is the optional data capture mongo config.
+type MongoConfig struct {
+	URI        string `json:"uri"`
```

Comment:
IIUC the idiomatic term for this is "MongoDB connection string" [1] - either in field name or documentation, we should refer as such

[1] https://www.mongodb.com/docs/manual/reference/connection-string/

Reply:
done

Reply:
oh I guess both work: https://www.mongodb.com/docs/drivers/go/current/fundamentals/connections/connection-guide/#connection-uri
![image](https://github.com/user-attachments/assets/44f39ae3-a61a-44d6-bb82-9e1f6b196fa3)

sorry! LGTM either way

Reply:
I'll change it back :)